embedded-graphics/src/primitives/triangle.rs

```diff
@@ -319,8 +315,14 @@ where
             match self.points() {
                 IterState::Border(point) => {
                     // Draw edges of the triangle
-                    if let Some(color) = self.style.stroke_color.or_else(|| self.style.fill_color) {
-                        if point.x >= 0 && point.y >= 0 {
+                    if point.x >= 0 && point.y >= 0 {
+                        if self.style.stroke_width > 0 {
+                            if let Some(color) = self.style.stroke_color {
```

Comment:
At first glance I thought both `return` statements would use the same `color`. By changing
```suggestion
                            if let Some(stroke_color) = self.style.stroke_color {
```
it would be immediately obvious that the colors are different.